dir.props

```diff
@@ -82,17 +83,31 @@
     <ValidatePackageVersions>true</ValidatePackageVersions>
     <ProhibitFloatingDependencies>true</ProhibitFloatingDependencies>
     <CoreFxExpectedPrerelease>rc3-24206-00</CoreFxExpectedPrerelease>
-    <CoreFxVersionsIdentityRegex>^(?i)((System\..*)|(NETStandard\.Library)|(Microsoft\.CSharp)|(Microsoft\.Private\.PackageBaseline)|(Microsoft\.NETCore.*)|(Microsoft\.TargetingPack\.Private\.(CoreCLR|NETNative))|(Microsoft\.Win32\..*)|(Microsoft\.VisualBasic))(?&lt;!TestData)(?&lt;!System\.Data\.SqlClient)(?&lt;!System\.IO\.Compression)$</CoreFxVersionsIdentityRegex>
+    <CoreClrExpectedPrerelease>rc3-24206-00</CoreClrExpectedPrerelease>
+    <ExternalExpectedPrerelease>rc3-24206-00</ExternalExpectedPrerelease>
+    <WCFExpectedPrerelease>rc3-24206-00</WCFExpectedPrerelease>
```

Comment:
The only way I know in which it matters is for package validation when our packages have a different version than corefx, but the only circumstance in which I have seen this be needed was just yesterday and it was because Wcf packages are still building out of TFS which hopefully won't be the case soon.